swarm/runtime/_claude_sdk/compat.py

```
@@ -0,0 +1,48 @@
+"""Compatibility shims for the Claude SDK adapter.
+
+This module provides backward compatibility helpers for gradual migration
+from older formats to newer standardized types.
+"""
+
+from typing import Any, Dict
```

Ruff (0.14.11)

`typing.Dict` is deprecated, use `dict` instead

(UP035)